src/ArduinoCloudProperty.hpp

```diff
@@ -18,34 +18,31 @@
 #ifndef ARDUINO_CLOUD_PROPERTY_HPP_
```

Comment:
A general suggestion - since `ArduinoCloudProperty` no longer is a template class it would make sense to rename the file to `ArduinoCloudProperty.h` since the file extension `.hpp` is generally used for files containing templates while `.h` is generally used for "normal" C++ classes.

Reply:
Done